include/robot_localization/navsat_transform.h

```diff
@@ -210,6 +210,10 @@ class NavSatTransform
     //!
     bool use_local_cartesian_;
 
+    //! @brief Whether we want to have our UTM zone fixed, and not change it if we move outside of it
```

Comment:
This sounds like an option. But I don't see where we can set the option :)

Reply:
Actually I wouldn't implement it as optional unless requested by a reviewer for backward compatibility. This will always lead to issues right?

Reply:
> Whether we want to have our UTM zone fixed

Is there any valid reason not to want it fixed?

Reply:
Ah, then yes, please rephrase the comment. This sounds like we can choose not to fix stuff.